src/tests/conformance/messages.rs

```diff
@@ -628,7 +714,7 @@ async fn get_data_blocks() {
     let mut node: Node = Default::default();
     node.initial_action(Action::SeedWithTestnetBlocks {
         socket_addr: new_local_addr(),
-        block_count: 3,
+        block_count: 2,
```

Comment:
Just making sure this isn't an accidental change?

Reply:
Good catch.